src/fiatExchanges/SimplexScreen.tsx

```diff
@@ -37,7 +37,7 @@ function SimplexScreen({ route, navigation }: Props) {
 
   const account = useSelector(currentAccountSelector)
   const e164PhoneNumber = useSelector(e164NumberSelector)
-  const phoneNumberConfirmed = useSelector(numberVerifiedSelector)
+  const phoneNumberConfirmed = useSelector(phoneNumberVerifiedSelector)
```

Comment:
this is kind of unrelated but i don't think Simplex should just be considering the decentralised status